bretm9/genrefy
# Nd styled gen playlist

### What’s this PR do?  
- PlaylistContainer now extends all of the available space and has overflow for when many GeneratedPlaylists are made.
- GeneratedPlaylist has more defined spacing and added borders for basic styling.
- Adds classes to certain sections to style them.
- Adds the artist name below the song name in GeneratedPlaylists.
 
### Where should the reviewer start?  
- Pull this branch
- GeneratedPlaylists directory, App.scss file, PlaylistContainer directory.
 
### How should this be manually tested?  
- run `npm start` and navigate to localhost:3000
- Check the styling on the page that the genres are to the left and the playlist container extends the rest of the screen on the right.
- When multiple playlists are added, or the data inside of a playlist extends beyond the screen, a user can scroll down to see additional songs and playlists that have been generated instead of pushing the page further down.
 
### Any background context you want to provide?  
- N/A
 
### What are the relevant tickets?  
- closes #91
 
### Screenshots (if appropriate)  
- N/A
 
### Questions: 
- N/A
